### DISCUSSION TITLE
[FEATURE REQ] Indexer Priority

### DISCUSSION THREAD

**Discussion by focheur91300** [autobrr/autobrr#1673] [Ideas]
Add priority to indexers, from 1 (highest) to 50 (lowest). Default value: 25. Used when entering versions to distinguish between equal versions, autobrr will always use all activated indexers for synchronisation.

**Comment by KyleSanderson** [comment]
What's the actual outcome you're looking to solve? the request doesn't make sense at face value.

**Comment by focheur91300** [comment]
I'm looking for autobrr to search on an indexer first. As is possible in Radarr, Sonarr. 
![image](https://github.com/autobrr/autobrr/assets/25317272/7b483d21-bb7c-450c-a57c-cc4be16d554a)

**Comment by KyleSanderson** [comment]
> I'm looking for autobrr to search on an indexer first. As is possible in Radarr, Sonarr. ![image](https://user-images.githubusercontent.com/25317272/261466531-7b483d21-bb7c-450c-a57c-cc4be16d554a.png)

That's not how Autobrr works though. Have you read the documentation on autobrr.com for upgraderr?

**Comment by zze0s** [comment]
To clarify on this: Prowlarr and the other arrs use RSS to find new media. It runs all the feeds and then checks for the monitored titles. If it's a match it will then grab the one with highest prio if there was results from multiple sources.

With autobrr you generally use it with IRC announces and if you are feeding arrs or directly to a client it will come from multiple sources at once. There is no queue etc since the point is to get things as fast as possible.

It may however be applicable for our feed (rss/torznab/newznab) implementations. Had not thought of that being useful before.
